### PR TITLE
Take action styles all screens

### PR DESCRIPTION
**Pull request by pdewouters**
- [x] check medium screen layout for steps
- [x] finish steps typography

### CODE REVIEW

File: src/scss/pages/_take-action.scss [hunk 1]
```diff
@@ -132,13 +161,33 @@
...
+  h2 {
     padding: 50px 0 0;
     color: $blue-heading;
   }
   .col-md-6 {
     padding-top: 30px;
   }
+  p {
+    color: $dark-para-text;
+    font-size: rem(17);
```
Comment (27Bslash6) [inline src/scss/pages/_take-action.scss]: 👍 Great to see a shift to rem values here.

These values are repeated in several places. Is it worth extracting these to global variables for re-use across the site to follow DRY principles, reduce cognitive load recalling standardised sizes, and adding semantic value?

File: src/scss/pages/_take-action.scss [hunk 2]
```diff
@@ -639,10 +655,51 @@
...
 @media (min-width: 1200px) {
+  .page-content-wrap {
+    padding-top: 200px;
+    .about-info {
+      margin-bottom: rem(72);
+    }
+  }
+
+  .what-you-can-do-wrapper  {
+    margin-bottom: 120px;
```
Comment (27Bslash6) [inline src/scss/pages/_take-action.scss]: Why is the .about-info margin in rem, but this in pixels?

File: src/scss/pages/_take-action.scss (hunk 1 above)
Comment (pdewouters) [reply src/scss/pages/_take-action.scss]: yes I agree and considered that, if there was a documented style guide that I could base that off, but at the moment, I have specific styles for a specific page, and the way the Sass is organized right now, would make it very time consuming to search for patterns across all files

**Comment by pdewouters** [comment]
@ashish173 I didn't commit the built stylesheet, you can rebuild that once this is merged into master

**Pull request merged**